RPS Project/Game.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import random
+
+score = [0,0,0]
+history = [0,0,0]
+count = [0]
```

Comment:
Is there a point to this being an array? Also, use a more descriptive name. `game_count` is probably enough.